demo/create.yml

```diff
@@ -1,11 +1,13 @@
 - name: Test Flight Control Collection to create resources
   hosts: localhost
+  vars: 
+    flightctl_config_file: "~/Library/Application\ Support/flightctl/client.yaml"
```

Comment:
```suggestion
```
This should no longer be necessary since you pass the value dynamically. I would probably add a note in the README.md file that you added, so that for OSX the path to the configuration file defaults to `“~/Library/Application/Application Support/flightctl/client.yaml”`.